README.md

```diff
@@ -1,4 +1,4 @@
-A minimal [RedJubjub][redjubjub] implementation for use in [Zebra][zebra].
+A minimal [RedDSA][reddsa] implementation for use in Zcash.
 
 Two parameterizations of RedJubjub are used in Zcash, one for
```

Comment:
Update this and other references to RedJubjub in this file.

Reply:
Yep, there is a bunch of this kind of cleanup work that we can do once there's agreement on the direction.